src/features/messages/cards/TransactionCard.tsx

```diff
@@ -236,11 +258,51 @@ function CallDataModal({ debugResult }: { debugResult?: MessageDebugResult }) {
           </p>
           <LabelAndCodeBlock label="Recipient contract address:" value={contract} />
           <LabelAndCodeBlock label="Handle function input calldata:" value={handleCalldata} />
+          <button onClick={handleClick} disabled={loading} className="underline text-blue-400">
```

Comment:
Show only when not loading (see feedback on discord).
Please also reduce text size: `text-sm`